packages/preferences/src/node/json-preference-server.ts

```diff
@@ -86,8 +92,9 @@ export class JsonPreferenceServer implements PreferenceServer {
                 }
                 return this.fileSystem.resolveContent(uri).then(({ stat, content }) => {
                     const strippedContent = jsoncparser.stripComments(content);
-                    return JSON.parse(strippedContent)
-                })
+
+                    return JSON.parse(strippedContent);
```

Comment:
Could you also investigate whether we can just use `parse` method of `jsonc-parser`: https://github.com/Microsoft/node-jsonc-parser#parser.

It seems to do the validation and tolerate bogus input, maybe we even don't need `ajv` at all.